kcmdline.c

```diff
@@ -53,10 +50,13 @@ EFI_STATUS check_cmdline(CONST CHAR8 *cmdline, UINTN cmdline_len) {
 	int start = -1;
 	int num_toks = 0;
 	buf = AllocatePool(cmdline_len + 1);
-	if (!buf) {
-		return EFI_OUT_OF_RESOURCES;
+	if (buf == NULL) {
+		goto out;
```

Comment:
This isn't right.  goto out followed by setting status?

Reply:
fixed that too. sorry.